contracts/upgradeable_contracts/BaseBridgeValidators.sol

```diff
@@ -102,4 +102,23 @@ contract BaseBridgeValidators is InitializableBridge, Ownable {
     function setNextValidator(address _prevValidator, address _validator) internal {
         addressStorage[keccak256(abi.encodePacked("validatorsList", _prevValidator))] = _validator;
     }
+
+    function isValidatorDuty(address _validator) external view returns (bool) {
```

Comment:
should we increase a version in `getBridgeValidatorsInterfacesVersion`?

Reply:
I think we should. Updated version to `2.3.0` https://github.com/poanetwork/tokenbridge-contracts/pull/319/commits/be3cd335b2bf2f4077759dfc0996e52e00b367a7